purchases/index.js

```diff
@@ -520,6 +520,22 @@ function listPurchases(db, options, callback) {
     });
 }
 
+function parseLimit({limit}){
+    const limitAsInt = parseInt(limit);
+    if(limitAsInt > 0 || !isNaN(limitAsInt)) {
+        return `LIMIT ${limitAsInt}`;
+    }
+    return "LIMIT ALL";
+}
+
+function parseOffset({offset}){
+    const offsetAsInt = parseInt(offset);
+    if(offsetAsInt > 0 || !isNaN(offsetAsInt)) {
+        return `OFFSET ${offsetAsInt}`;
+    }
+    return "OFFSET 0";
```

Comment:
Same as above: could be empty string.

Reply:
The advantage of using empty string here is that it is immediately clear to reviewers that passing no options keeps the generated SQL *exactly* equivalent to what was there before.